settings/.env.example

```diff
@@ -1,7 +1,8 @@
 RESOURCES_FOLDER="resources"
 USER_MESSAGES_FOLDER="user-messages"
 DRIVERS_FOLDER="drivers"
-IMG_FOLDER="img"
+IMG_FOLDER_DEFAULT="img"
```

Comment:
Why having IMG_FOLDER_DEFAULT when you have IMG_ABS_PATH? IMO it is redundant, with IMG_ABS_PATH is enough.